conan/tools/cmake/file_api.py

```diff
@@ -70,18 +68,6 @@ def build_type(self):
         """
         return self._conanfile.settings.get_safe("build_type")
 
-    class CodeModelQueryV2(object):
```

Comment:
A bit unnecessary at this point, add types, classes, etc only when really needed.